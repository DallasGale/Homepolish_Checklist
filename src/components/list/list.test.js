import React from 'react';
import List from './list.component';
import renderer from 'react-test-renderer';

const tasks = [];

describe('<List />', () => {
  
    it('will render without exploading!', () => {
        const tree = renderer.create(<List data={ tasks } />).toJSON();
        
        expect(tree).toMatchSnapshot();
    });

})