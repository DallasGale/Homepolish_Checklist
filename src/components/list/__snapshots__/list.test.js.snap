// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> will render without exploading! 1`] = `
<div
  className="sc-EHOje jNRjeF"
>
  <ul
    className="sc-bZQynM ebnWYe"
  />
</div>
`;
